test/utils/image/manifest.go

```diff
@@ -77,8 +77,8 @@ var (
 	Net                      = ImageConfig{e2eRegistry, "net", "1.0", true}
 	Netexec                  = ImageConfig{e2eRegistry, "netexec", "1.0", true}
 	Nettest                  = ImageConfig{e2eRegistry, "nettest", "1.0", true}
-	NginxSlim                = ImageConfig{gcRegistry, "nginx-slim", "0.20", true}
-	NginxSlimNew             = ImageConfig{gcRegistry, "nginx-slim", "0.21", true}
+	Nginx                    = ImageConfig{"", "nginx", "1.14-alpine", false}
```

Comment:
I don't think this will work for cri-o (it doesn't default to docker hub). Can we check with sig-node?

Reply:
@cblecker This will frame imagename: `nginx:1.14-alpine` why do you think it will not work with cri-o?

Reply:
@mkumatag I'm trying to find the issue, but IIRC, there was an issue at some point with us assuming that if we leave off the registry that we would pull from docker hub (docker being not the only container runtime we support).

Either way, what wouldn't hurt is to fully qualify the image.

If we created a constant of `dhRegistry = "index.docker.io/library"` and then use that here, it should fully qualify the image name.

Reply:
I did that intially, but in docker runtime it is creating an issue.. after the deploymemt it wasn't matching the name with registry name.. so then added this logic of not having registry name for docker hub..

Reply:
@cblecker looks like CRI-O has support for specifying registries when registry names are unqualified - https://github.com/kubernetes-incubator/cri-o/pull/674/commits/a35727c80bd2a26613aae21db00628045cb9be24

@mkumatag they seem to be using `docker.io` ... so can we do `dhRegistry = "docker.io"`? what failure were you seeing when you use that?

Reply:
@mkumatag - the following seems to work right?

```
dims@bigbox:~$ docker --debug run -i -t docker.io/ubuntu /bin/bash
root@5053bfa04c09:/#
```

Reply:
we should probably use `docker.io/library` instead of just `docker.io`.. the registry URL is technically `https://registry-1.docker.io/v2/library/nginx/manifests/latest`

Reply:
I don't remember the errors what I faced, let me put this logic and see how it goes.

@cblecker I prefer using `docker.io` because this is the actual registry name, and the library is just a namespace which we can ignore or specify with imagename if required.